modal/sandbox.py

```diff
@@ -425,6 +426,22 @@ async def exec(self, *cmds: str, pty_info: Optional[api_pb2.PTYInfo] = None):
         )
         return _ContainerProcess(resp.exec_id, self._client)
 
+    async def open(self, path: str, mode: str = "r") -> _FileIO:
```

Comment:
Can we support the context manager api too?

Reply:
It should already be supported